api/rest/swagger.yaml

```diff
@@ -0,0 +1,796 @@
+      tags:
+        - peers
+      description: Add a new Peer to the Cluster
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                peer_id:
+                  type: string
+            example: {"peer_id":"QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB"}
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Peer'
+              example: {"addresses":["/ip4/192.168.86.35/tcp/9196/ipfs/QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","/p2p-circuit/ipfs/QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","/ip4/127.0.0.1/tcp/9196/ipfs/QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","/ip4/10.6.0.2/tcp/9196/ipfs/QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","/ip4/192.168.1.42/tcp/9196/ipfs/QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","/ip4/10.0.3.1/tcp/9196/ipfs/QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB"],"cluster_peers":["QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH"],"cluster_peers_addresses":["/ip4/127.0.0.1/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/10.6.0.2/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/10.71.2.85/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/10.0.3.1/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/192.168.1.42/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/ip4/192.168.86.35/tcp/9096/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH","/p2p-circuit/ipfs/QmSJz3VA8cQVeegqZGsTkKQ4uiBXzWBE4r43KPb8woZbtH"],"commit":"","error":"","id":"QmQnMiKwmxpHd3RizCk2Gk58TFp6d5n9pd9FXRnPFZiAKB","ipfs":{"addresses":["/ip4/127.0.0.1/tcp/4002/ipfs/Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw","/ip4/192.168.86.35/tcp/4002/ipfs/Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw","/ip4/10.0.3.1/tcp/4002/ipfs/Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw","/ip4/10.6.0.2/tcp/4002/ipfs/Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw","/ip4/192.168.1.42/tcp/4002/ipfs/Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw","/ip6/::1/tcp/4002/ipfs/Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw"],"error":"","id":"Qmb1ThT2zJ1MVcqboqh6KwtZQ3TT8kEXeV6fUPJRVVZEqw"},"peername":"omou","rpc_protocol_version":"/ipfscluster/0.10/rpc","version":"0.10.0"}
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              example: {"code":400,"message":"error decoding peer_id"}
+              
+  /peers/{peer}:
+    parameters:
+      - name: peer
+        in: path
+        required: true
+        schema:
+          type: string
+    delete:
+      summary: Remove a Peer from the Cluster
+      tags:
+        - peers
+      description: Remove a Peer from the Cluster
```

Comment:
`This endpoint will attempt to remove a peer from a Raft-based cluster. If the removed peer is running, it will automatically shutdown. The operation can only succeed if a Raft leader exists. This operation has no  effect for CRDT-based Clusters.`